src/components/sidebar/sidebar.module.css

```diff
@@ -16,3 +16,41 @@
   margin: 0;
   padding: 0;
 }
+
+/*
+***
+* Classes for the mobile sidebar's level button
```

Comment:
`SidebarBackToLink` will also need classes like this. Thinking about housing both the `StandaloneLink` and `Button` versions of the "back to element" into one file, that way they can share classes or at least the complex logic of "spacing is different depending on what comes after".

Reply:
Removed, no longer relevant! ✨